spec/requests/physical_servers_spec.rb

```diff
@@ -10,6 +10,155 @@
         expect_single_resource_query("ems_ref" => "A59D5B36821111E1A9F5E41F13ED4F6A")
       end
     end
+
+    context "without an appropriate role" do
```

Comment:
Generally prefer context blocks when they are used to establish some shared setup through before blocks, lets, etc.. I think it's OK to put this "context" in test description itself